src/cli_plugin/install/install.js

```diff
@@ -30,8 +30,6 @@ export default async function install(settings, logger) {
 
     await renamePlugin(settings.workingPath, path.join(settings.pluginDir, settings.plugins[0].name));
 
-    await rebuildCache(settings, logger);
```

Comment:
What if we kept `rebuildCache` as the default, but added a `skip-optimize` or something like that would skip rebuild? Then users could opt into this. We could also document this as the "recommended approach", but we can't make it a default just yet because it would be a breaking change.

Reply:
Thoughts on taking a separate approach for 6.x (if any)?  I'm fine targeting this for 7.0 only even if it's getting removed.  

I ask because we do have workarounds documented, but there's so many bin/plugin install guides that's its usually not followed.

Reply:
Part of the reason I'm so heavy handed in this PR is I'm not sure why we bundle on plugin install.  I've had suspicions that we went this way on v1 and never revisited, but if there's more context I'd love to hear it.

Reply:
Targeting this for K7 is okey with me (even if that means making no changes to 6.x).

Reply:
Let's do --skip-optimize for 6.x and --optimize for 7.0.  Just make sure to add it to the breaking changes in master.  And we should update all references in our docs to use --skip-optimize to help folks migrate to this pattern sooner.

Reply:
Also, if you do what I just suggested, I strongly recommend making one PR that makes the --skip-optimize change to both master/6.x, then follow up with a PR that adds the --optimize argument, breaking change doc, and a clear error message when someone uses --skip-optimize